sigle/src/modules/settings/SettingsForm.tsx

```diff
@@ -171,21 +216,54 @@ export const SettingsForm = ({ settings, username }: SettingsFormProps) => {
     multiple: false,
   });
 
-  const handleRemoveCover = (
```

Comment:
So, one can not remove the image anymore, is it the desired behavior?

Reply:
will check this over with Quentin 👍